crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -1321,6 +1321,9 @@ async fn test_handle_transfer_transaction_with_max_sequence_number() {
         rgp * TEST_ONLY_GAS_UNIT_FOR_TRANSFER,
         rgp,
     );
+    let transfer_transaction = authority_state
```

Comment:
nit: I'd pass `authority_state` to `init_transfer_transaction` and make it return `VerifiedTransaction`